service.yml

```diff
@@ -73,7 +73,6 @@ code_artifact:
   enable: true
   package_paths:
     - maven-snapshots/maven/io.confluent.kafka-images/kafka-images-parent
-    - maven-snapshots/maven/io.confluent.kafka-images/cp-zookeeper
```

Comment:
hey, we can't remove this package path, as we still need zookeeper to be built for CP 7.x branches, had checked with devprod team on the config https://confluent.slack.com/archives/C09EP1SS3/p1741157167054199